### DISCUSSION TITLE
Having some trouble getting shark badge points

### DISCUSSION THREAD

**Discussion by jessemchung** [community/community#138829] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello, sorry to bother, but I'm just curious, recently I've been trying to contribute to open source projects and I've been having some confusion with getting the pull shark badge.  I'm not sure if I'm doing something wrong or if this just doesn't count.  It is a very small pull request.

This was the pull request:
https://github.com/zaproxy/zaproxy/pull/8630
<img width="1792" alt="Screenshot 2024-09-16 at 2 13 37 PM" src="https://github.com/user-attachments/assets/327ad4cf-f369-45f9-a0e1-126a50201c71">

But sadly, no badge?
<img width="926" alt="Screenshot 2024-09-16 at 2 14 19 PM" src="https://github.com/user-attachments/assets/3e4c92ef-a758-4546-b347-7656a9a8cd97">

**Comment by barishaxxer** [comment, marked as answer]
You get the Pull Shark badge 3x when your 128th PR is merged. Are you sure this was your 128th pull request that was merged? Because everything seems to be correct.

**Reply by jessemchung** [reply]
Oh wait, I think I see now. I guess I am just confused because when I click the badge, I cannot see my total prs merged. So it looked like to me that it said x2 because I have 2 pull requests listed. Now I understand. Is there a way to see how far along I am though to getting the next badge?

**Comment by davevad93** [comment]
Hi @jessemchung ,

To see the total count of your PRs, go to your profile [pulls page](https://github.com/pulls), click on `Closed` and you'll see a list of your closed PRs; for the `Pull Shark` achievement only your merged PRs are relevant.

![Captura de pantalla 2024-09-16 225927](https://github.com/user-attachments/assets/9c1fcb9a-9ca6-4b50-9ccf-45e388cf308a)

**Reply by jessemchung** [reply]
Oh I see how this works now.  Interesting.  So you can actually get both pair extraordinaire and pull shark points with the same pull request.  Thanks

**Reply by davevad93** [reply]
You're welcome @jessemchung , yes you can get both achievement points with the same PR! 👍
